### PR TITLE
Personal Information tab on Settings(change username and email)

### PR DESCRIPTION
**Pull request by coffeefee-coder**

### CODE REVIEW

File: views/settings.html [hunk 1]
```diff
@@ -15,7 +15,52 @@
       </ul>
     </div>
     <div id="information" class="col s12 m12 l12">
-      Personal Information
+      <form class="row s24" name="changeUsername">
```
Comment (sonph) [inline views/settings.html]: No need for class `s24` here!

File: views/settings.html [hunk 2]
```diff
@@ -15,7 +15,52 @@
...
+        <h5>Change Username</h5>
+        <div class="row">
+          <div class="input-field col s24 m6 16">
```
Comment (sonph) [inline views/settings.html]: Class should be `l6` (as in Large) instead of `16`

File: views/settings.html [hunk 3]
```diff
@@ -15,7 +15,52 @@
...
+                ng-required="true">
+            <label class="active" for="username">Username</label>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col s12 m12 l12">
+            <input
+              type="submit"
+              value="Change Username"
+              class="waves-effect waves-light btn lighten-1 white-text"
```
Comment (sonph) [inline views/settings.html]: You can add class `blue` so the button color is blue instead of default green.

File: views/settings.html [hunk 4]
```diff
@@ -15,7 +15,52 @@
...
+              ng-click="change_username()">
+          </div>
+        </div>
+      </form>
+      <form class="row s24" name="changeEmail">
```
Comment (sonph) [inline views/settings.html]: Same here

File: views/settings.html [hunk 5]
```diff
@@ -15,7 +15,52 @@
...
+        <h5>Change Email</h5>
+        <div class="row">
+          <div class="input-field col s24 m6 16">
```
Comment (sonph) [inline views/settings.html]: Same here, `l6` instead of `16`

File: views/settings.html [hunk 6]
```diff
@@ -15,7 +15,52 @@
...
+                ng-required="true">
+            <label class="active" for="email">Email</label>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col s12 m12 l12">
+            <input
+              type="submit"
+              value="Change Email"
+              class="waves-effect waves-light btn lighten-1 white-text"
```
Comment (sonph) [inline views/settings.html]: Add class `blue` to the list.

**Comment by sonph** [comment]
Looks good other than a couple of inconsistencies! :+1: :+1:

**Comment by coffeefee-coder** [comment]
Alright I think I've taken care of all the things you commented on!

**Comment by sonph** [comment]
Cool :+1:

**Comment by Spencatro** [comment]
Things to do before merge:
- Remove ability to edit username
- Adding names

**Pull request merged**